templates/duplicateto_select.mustache

```diff
@@ -29,10 +29,8 @@
     }
 }}
 <select id="block-massaction-control-section-list-duplicateto"
-        name="{{#str}} action_duplicatetosection, block_massaction {{/str}}"
-        aria-label="{{#str}} action_duplicatetosection, block_massaction {{/str}}">
```

Comment:
A `<select>` often needs a <label for="block-massaction-control-section-list-duplicateto">Day of the Week<label>. Do you know if one is provided at the higher level? Otherwise, aria-label may be a viable alternative for screen readers as long as sufficient context is visually available.

The "name" field is what the browser uses as the key when submitting a form. You may not want to use a translated string there as the code may need to access the data by key.

Reply:
In this case we do not have a label because of the layout decisions. The first `option` (and shown by default) really is the label here. I just tested with NVDA screenreader, it properly announces this "label" on mouse over which I guess should be fine.

`name` field: I only added this for the behat tests to be honest. The code extracts the selected information via JS and does some transformations before putting it into a hidden field which then is being submitted by the form. Would you propose to change that to a non translated string?

Reply:
> In this case we do not have a label because of the layout decisions. The first `option` (and shown by default) really is the label here. I just tested with NVDA screenreader, it properly announces this "label" on mouse over which I guess should be fine.

Functionally (for a human), that may be fine. Automated accessibility checkers, like the [WAVE browser extension](https://wave.webaim.org/extension/), may flag it as an error. Although the accessibility checkers are far less important than a human's experience, we tend to try and keep both happy. 😁 

> `name` field: I only added this for the behat tests to be honest. The code extracts the selected information via JS and does some transformations before putting it into a hidden field which then is being submitted by the form. Would you propose to change that to a non translated string?

Yes, if the `name` attribute is only present for Behat tests, then it's even more important that the `name` attribute be a static (non-translated) computer-friendly string. That would potentially avoid Behat tests failing for users who are running tests against a site that is using language strings that do not match the default.

Reply:
Thank you very much for the feedback. Would you mind having a look at #36 ?